api/test/e2e/journeys/test_subjects.py

```diff
@@ -92,5 +93,95 @@ def test_list_all_subjects(self):
             ]
         )
 
-    # TODO: Test the create/delete client flow
-    # TODO: Test the create/delete user flow
+    def test_create_delete_user_flow(self):
```

Comment:
Does it make sense to make these two separate tests? I'm honestly not sure tbh

Reply:
Not strongly convinced either; they read fine as is, so will leave for now